frontend/src/components/WeaponIcon.vue

```diff
@@ -88,6 +88,8 @@ const gColor = new Three.Color(0x413F41);
 const bColor = new Three.Color(0xAF5822);
 const white = new Three.Color(0xFFFFFF);
 
+const censor = new CensorSensor();
```

Comment:
can this get abstracted to a separate service or somewhere else it can be imported? like the getWeaponNameFromSeed function etc. Then we have one function we can call everywhere instead of initializing this everywhere. If we want to update the profanity we then also have one file we have to do so.